packages/start-slicemachine/src/StartSliceMachineProcess.ts

```diff
@@ -115,12 +115,14 @@ export class StartSliceMachineProcess {
 		} catch {
 			// noop - We'll try again before taking a screenshot.
 		}
-		await Promise.all([
-			profile
-				? this._sliceMachineManager.user.refreshAuthenticationToken()
-				: undefined,
-			profile ? this._sliceMachineManager.screenshots.initS3ACL() : undefined,
-		]);
+		if (profile) {
+			await Promise.allSettled([
+				// noop - We'll try again when needed.
```

Comment:
Sure! Here we're performing two things:

1. Refreshing the user auth token to keep its session active longer
2. Initing its S3 ACL for the current repository

Both those operations yield promises.

If 1. fails, it's not a big deal, the user session might expire earlier

If 2. fails, it's not a big deal either, it might be because the user doesn't have access to this repository S3 ACL, in any case, this will be attempted again before uploading screenshots. The reason we're doing it here is to attempt to later speed up the screenshot upload process by having this step already performed.

---

There is no `try`/`catch` because `allSettled` catches and returns promise results whether they resolve or reject: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled#return_value

I used `allSettled` over `all` here in case we need to retrieve the promise results and analyze them.

---

Let me know if anything :)